### PR TITLE
Early Review: Running Analysis

### PR DESCRIPTION
**Pull request by gbdubs**
Looking for early feedback here - does this look like a reasonable request structure, and if so, have I interpreted how to run the report correctly? I want validation of that before I head too far down this road!

### CODE REVIEW

File: cmd/runner/main.go [hunk 1]
```diff
@@ -356,28 +358,46 @@ func countCSVLines(path string) (int, error) {
...
+		return nil, fmt.Errorf("failed to load CreateReportRequest: %w", err)
 	}
+	return &task, nil
+}
 
-	return &task.CreateReportRequest{
-		PortfolioID: pacta.PortfolioID(pID),
-	}, nil
+func (h *handler) createAudit(ctx context.Context, taskID task.ID, req *task.CreateAuditRequest) error {
+	return fmt.Errorf("not implemented")
```
Comment (bcspragu) [inline cmd/runner/main.go:386]: niiit

```suggestion
	return errors.New("not implemented")
```

Normally `nogo` should pick stuff like this up, have we disabled it on this project?

**Review by bcspragu — COMMENTED** [review]
Structurally LGTM, just small comments and pointers

File: cmd/runner/main.go (hunk 1 above)
Comment (gbdubs) [reply cmd/runner/main.go:386]: I don't think nogo is set up for this project. I didn't find top level references to it. I'll add a Bug to track adding it.

Comment (gbdubs) [reply cmd/runner/main.go:386]: #115

File: azure/azevents/azevents.go [hunk 2]
```diff
@@ -44,9 +46,16 @@ type DB interface {
...
+
+	Analysis(tx db.Tx, id pacta.AnalysisID) (*pacta.Analysis, error)
+	UpdateAnalysis(tx db.Tx, id pacta.AnalysisID, mutations ...db.UpdateAnalysisFn) error
+
+	CreateAnalysisArtifact(tx db.Tx, a *pacta.AnalysisArtifact) (pacta.AnalysisArtifactID, error)
 }
 
 const parsedPortfolioPath = "/events/parsed_portfolio"
+const createdAuditPath = "/events/created_audit"
+const createdReportPath = "/events/created_report"
```
[the remark this answers is not in the document]
Comment (gbdubs) [reply azure/azevents/azevents.go:58]: GC will send a PR.

[the remark this answers is not in the document]
Comment (gbdubs) [reply azure/azevents/azevents.go:58]: [Filed #22](https://github.com/RMI/terraform/issues/22)

**Comment by bcspragu** [comment]
I had hideous troubles trying to merge the recent changes into this branch, so I ended up rebasing it, and instead of force-pushing to this branch, I just created a new one at https://github.com/RMI-PACTA/app/compare/brandon/runanalysis2?expand=1, where I'll keep working on this.

Closing this out

**Pull request closed without merging**